src/jitlayers.cpp

```diff
@@ -1282,6 +1282,53 @@ JuliaOJIT::JuliaOJIT()
     // symbols in the program as well. The nullptr argument to the function
     // tells DynamicLibrary to load the program, not a library.
     std::string ErrorStr;
+
+    const char *aliasIR = R"V0G0N"(
```

Comment:
I would use the LLVM C++ API to codegen these

Reply:
Yeah, there are probably cleaner ways of doing this for sure. This is just to know if it works.
Even if textual IR is the final thing it probably should go in a different file or something